packages/story-editor/src/app/story/storyTriggers/storyEvents/propTypes.js

```diff
@@ -17,7 +17,7 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import { OutputStory as storyPropType } from '@googleforcreators/output'; // @todo See issue #11015
+import { OutputStory as storyPropType } from '@googleforcreators/output';
```

Comment:
are we using this file anywhere anymore? should we just delete this file if not?

Reply:
Looks like it's still getting used through `getStoryMarkup` which is in turn used in the `wp-story-editor` for status check as well as in the dashboard.

Reply:
These proptypes were only used in the two registers, `onInitialElementAddRegister` and `onPageAdded` so I removed this file since I updated those propTypes locally.